cloudcopy/server/tasks/workflow.py

```diff
@@ -0,0 +1,172 @@
+import json
+import asyncio
+import sys
+
+import aiofiles
+from adbc.workflow import Workflow as Runner
+
+from cloudcopy.server.tasks.core import app
+from cloudcopy.server.utils import get_uuid, now
+from cloudcopy.server.storage import get_internal_database
+from cloudcopy.server.models import Job, Workflow
+from cloudcopy.server.config import settings
+
+
+class WorkflowLogger(object):
+    def __init__(self, verbose=False, stdout=None):
+        self.verbose = verbose
+        self.stdout = stdout or sys.stdout
+
+    def setLevel(self, level, *args):
+        self._level = level
+
+    def log(self, *args, **kwargs):
+        if verbose and self.stdout != sys.stdout:
+            print(*args)
+        self.stdout.write(*args)
+        if hasattr(self.stdout, 'flush'):
+            self.stdout.flush()
+
+    def info(self, *args, **kwargs):
+        return self.log(*args, **kwargs)
+
+    def debug(self, *args, **kwargs):
+        return self.log(*args, **kwargs)
+
+
+async def _execute(workflow_id: str):
```

Comment:
Execute a workflow
Assumption is that this method will be scheduled to fire based on the workflow's schedule, or manual trigger by: `POST /v0/workflows/:ID/trigger`